_faq/application-template.md

```diff
@@ -1,8 +1,14 @@
 ---
 question: "Do you have an application template?"
 ---
-coala 2018 Application Template
+{% assign month = "now" | date: "%-m" | plus:0 %}
+{% if month >= 9 %}
+coala {{ "now" | date: "%Y" | plus:1 }} Application Template
 ===============================
+{% else %}
+coala {{ "now" | date: "%Y" }} Application Template
+===============================
```

Comment:
this line doesnt need to be inside the `if`

Reply:
where else would it go?

Reply:
Outside the else block, after `end if`

Reply:
then you'd have double headings

Reply:
No, John meant moving `======` outside if

Reply:
Tried that and it doesn't render as a heading unless the title and `======` are on adjacent lines. Simply replaced it with `#` heading 1 tags which format the heading in the same way as `======`.